### PR TITLE
Add validator for any IP address

### PR DESCRIPTION
**Pull request by jdevesa**
Provide a validator for IP addresses, regardless they are IPv4 or IPv6

### CODE REVIEW

**Comment by bmjen** [comment]
Thanks @jdevesa. This looks pretty good. Can you add docs for this new function?

**Comment by strider** [comment]
:+1:

**Comment by jdevesa** [comment]
@bmjen Thanks!. Take a look at the second commit with the docs and tell me if is good enough, please

File: README.markdown [hunk 1]
```diff
@@ -1230,3 +1263,4 @@ To report or research a bug with any part of this module, please go to
 ##Contributors
 
 The list of contributors can be found at: https://github.com/puppetlabs/puppetlabs-stdlib/graphs/contributors
+
```
Comment (bmjen) [inline README.markdown]: Can you remove this extra newline?

**Comment by bmjen** [comment]
Thanks @jdevesa. Things look great. Can you make that last tiny edit and squash these 2 commits down to 1 and we look good to go!

**Comment by jdevesa** [comment]
Sure @bmjen

**Pull request merged**